### DISCUSSION TITLE
Location sharing

### DISCUSSION THREAD

**Discussion by ogezue** [teslamate-org/teslamate#738] [Ideas]
It would be great if you can add a location-sharing feature.
Use case: I drive abroad where I have high roaming fees or no internet on my mobile. I want to be able to share my location with certain people for a specific duration. So they know, when I arrive.

**Comment by JakobLichterfeld** [comment]
Very cool idea especially for regions without romaing for no charge. Teslamte is a data logger, not an automation service nor an sharing service. Latitude and longitude are already exposed as a MQTT topic, so relativly easy to use with external apps. If you build an app for it, just link it in the docu :-)
